src/test/java/io/confluent/kafka/connect/datagen/DatagenTaskTest.java

```diff
@@ -155,15 +157,106 @@ public void shouldGenerateFilesForTransactionsQuickstart() throws Exception {
   public void shouldGenerateFilesForStoresQuickstart() throws Exception {
     generateAndValidateRecordsFor(Quickstart.STORES);
   }
+  
   @Test
   public void shouldGenerateFilesForCampaignFinanceQuickstart() throws Exception {
     generateAndValidateRecordsFor(Quickstart.CAMPAIGN_FINANCE);
   }
 
+  @Test
+  public void shouldGenerateFilesForFleetMgmtDescriptionQuickstart() throws Exception {
```

Comment:
nit: would help if you sort the tests by name , then matching test to file names becomes easy

Reply:
@garrix-fan why don't we use a dataprovider for these tests?

Reply:
@shaikzakiriitm That can be done with Junit 4 [Parameterized Tests](https://github.com/junit-team/junit4/wiki/Parameterized-tests) (similar but not same as TestNg DataProviders), but we would have to enclose two sets of tests (quickstart and others) in different static classes by making use of [Enclosed Runners](https://github.com/junit-team/junit4/wiki/%27Enclosed%27-test-runner-example) which is an experimental feature in Junit4. Another option is to create a new test class only for testing quickStart templates. 
PS: There is also an option to loop for each quickstart enum and test with same method but we would have to handle `@Before` and `@After` ourselves adding an inconsistency.

Reply:
Okay, TestNg dataprovider pattern looks more readable to me mainly because we can have dataprovider attached to a single test method unlike in junit where we need to attach dataprovider. wdyt? Other option is to upgrade to junit5 for running these parametrized tests, that way same class can have both parameterized tests as well as non-parameterized tests.

Reply:
We can upgrade to junit5. Do you think that should be done in another PR?